test_results_win64.csv

```diff
@@ -120,26 +120,26 @@ Generic System,All Equity Partnership Flip,756864000,32.812180,36.422478,;
 Generic System,Sale Leaseback,756864000,32.130512,35.256783,; 
 Generic System,LCOE Calculator,756864000,NA,NA,; 
 Generic System,None,756864000,NA,NA,; 
-PV Battery,Residential,8588.589844,11.205457,NA,; 
-PV Battery,Commercial,911877.375000,4.274710,NA,; 
-PV Battery,Third Party,34803.335938,NA,NA,; 
-PV Battery,Host Developer,911877.375000,13.250628,10.808079,; 
-PV Battery,Single Owner,92853624,7.822259,9.770245,; 
-PV Battery,Merchant Plant,92853712,6.944926,NA,; 
-PV Battery,Leveraged Partnership Flip,92853712,7.926886,9.770247,; 
-PV Battery,All Equity Partnership Flip,92853712,9.339341,10.855831,; 
-PV Battery,Sale Leaseback,92853712,9.509977,9.770247,; 
-PVWatts Battery,Residential,8022.692383,11.707509,NA,; 
+PV Battery,Residential,8588.509766,11.206270,NA,; 
+PV Battery,Commercial,911876.750000,4.274711,NA,; 
+PV Battery,Third Party,34803.347656,NA,NA,; 
+PV Battery,Host Developer,911876.750000,13.250633,10.808079,; 
+PV Battery,Single Owner,92853440,7.794683,9.771930,; 
+PV Battery,Merchant Plant,92853536,6.917604,NA,; 
+PV Battery,Leveraged Partnership Flip,92853536,7.926891,9.771934,; 
+PV Battery,All Equity Partnership Flip,92853536,9.312499,10.857704,; 
+PV Battery,Sale Leaseback,92853536,9.510679,9.771934,; 
+PVWatts Battery,Residential,8022.691406,11.707511,NA,; 
```

Comment:
Are these value changes expected when using the NMC Lifetime model instead of the Calendar-Cycle definitions?